__tests__/items-container.test.tsx

```diff
@@ -334,71 +711,244 @@ describe('ItemsContainer component behaviour', () => {
+        expect(objectData[0].order).toEqual(2);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(0);
+        objectData.sort(componentWrapper.instance().compare('string', true));
+        expect(objectData[0].order).toEqual(0);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(2);
+
+        expect(objectData).toEqual(sortingTestData);
+    });
+
+    test('Verify sort method handles contentType:password', () => {
+        componentWrapper = manyWhoMount();
+        const objectData = Object.assign(sortingTestData);
+
+        objectData.sort(componentWrapper.instance().compare('password', false));
+        expect(objectData[0].order).toEqual(2);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(0);
+        objectData.sort(componentWrapper.instance().compare('password', true));
+        expect(objectData[0].order).toEqual(0);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(2);
+
+        expect(objectData).toEqual(sortingTestData);
+    });
+
+    test('Verify sort method handles contentType:datetime', () => {
+        componentWrapper = manyWhoMount();
+        const objectData = Object.assign(sortingTestData);
+
+        objectData.sort(componentWrapper.instance().compare('datetime', false));
+        expect(objectData[0].order).toEqual(2);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(0);
+        objectData.sort(componentWrapper.instance().compare('datetime', true));
+        expect(objectData[0].order).toEqual(0);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(2);
+
+        expect(objectData).toEqual(sortingTestData);
+    });
+
+    test('Verify sort method handles contentType:number', () => {
+        componentWrapper = manyWhoMount();
+        const objectData = Object.assign(sortingTestData);
+
+        objectData.sort(componentWrapper.instance().compare('number', false));
+        expect(objectData[0].order).toEqual(2);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(0);
+        objectData.sort(componentWrapper.instance().compare('number', true));
+        expect(objectData[0].order).toEqual(0);
+        expect(objectData[1].order).toEqual(1);
+        expect(objectData[2].order).toEqual(2);
+
+        expect(objectData).toEqual(sortingTestData);
+    });
+
+    test('Verify sort method handles contentType:boolean', () => {
+        componentWrapper = manyWhoMount();
+        const objectData = Object.assign(sortingTestData);
+
+        objectData.sort(componentWrapper.instance().compare('boolean', false));
+        // Only two unique values so the result of sorting three rows is non-deterministic
+        expect(objectData[0].properties[0].contentValue).toEqual('');
```

Comment:
Maybe try having one boolean being True, one being False, and one being empty.
Having no False and 2 True seems like we're missing some edge cases.

Reply:
Will do